src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -219,4 +219,15 @@ public static int getSampleFrequencyFromLibMediaInfo(String mediaInfoValue) {
 			return result;
 		}
 	}
+
+	@Override
+	public String toString() {
+		StringBuilder result = new StringBuilder();
+		result.append("Channel(s): ").append(getNumberOfChannels());
+		if (getAudioDelay() != 0) {
+			result.append(", Delay: ").append(getAudioDelay());
+		}
+
```

Comment:
@Nadahar I will check that.

As thought that i asked you about that and didn't get answer, but maybe i didn't really send my PR annotation, perhaps they has been only pending and deleted later by me...